specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2023-02-01/keys.json

```diff
@@ -624,6 +624,7 @@
             "modelAsString": false,
             "values": [
               {
+                "name": "RotateHSM",
```

Comment:
- Before the fix, python sdk has [duplicated enum](https://github.com/azure-sdk/azure-sdk-for-python/pull/4572/files#diff-f7d05b16c1c3690a6ae1065dc6e072d7af5cd502bd739422c1448d26b8a36cd1):
![image](https://github.com/Azure/azure-rest-api-specs/assets/70930885/fc095715-c2d1-44a5-9310-186264cc06e6)
- After the fix, there is [no duplicated enum](https://github.com/azure-sdk/azure-sdk-for-python/pull/4577/files#diff-f7d05b16c1c3690a6ae1065dc6e072d7af5cd502bd739422c1448d26b8a36cd1):
![image](https://github.com/Azure/azure-rest-api-specs/assets/70930885/f7462878-f9d6-4825-b524-a0ad43308be3)